app/src/org/commcare/android/view/TabbedDetailView.java

```diff
@@ -37,8 +38,10 @@
     private EntityDetailPagerAdapter mEntityDetailPagerAdapter;
     private ViewPager mViewPager;
     private View mViewPagerWrapper;
-    
+
     private int mAlternateId = -1;
+
+    private boolean useNewTabStyle = true;

```

Comment:
Might be good to make this USE_NEW_TAB_STYLE and final, so that it is clear that it is a hand set flag.